sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -491,6 +491,18 @@ static void rt1308_init(const struct snd_soc_acpi_link_adr *link,
 	}
 }
 
+static void rt715_init(const struct snd_soc_acpi_link_adr *link,
+		       struct snd_soc_dai_link *dai_links,
+		       struct codec_info *info,
+		       bool playback)
+{
+	/*
+	 * dai id is fixed at 4 for 715 to keep sdw
+	 * dmic and hdmi setting static in ucm
+	 */
+	dai_links->id = 4;
```

Comment:
If this has to agree with UCM, this is a part of the ABI, right? Then we might need some macros for all such values...

Reply:
thanks, I will refine it